### ISSUE TITLE
Collapsed Layout?

### ISSUE THREAD

**Issue by Exerosis** [cachapa/ExpandableLayout#35]
It would be really helpful to support a collapsed layout as well for situations where you want to show some content, then expand to show more. In my case the description of an article.

**Comment by cachapa** [comment]
Do you mean showing a bit of the layout in the collapsed state?
I guess that would be possible by configuring the collapsed size, but honestly I struggle to find a common use case for it.

I'll leave this issue open for now.

**Comment by Exerosis** [comment]
Yes, I think it could be helpful for anyone looking to have a preview of something(image, text, comments in a list, etc). Something similar to a minHeight would be really useful! Thanks for considering.

**Comment by powerhcm8** [comment]
I've done was like this:

     <LinearLayout>//root
         <LinearLayout/>//child
         <net.cachapa.expandablelayout.ExpandableLayout/>
     </LinearLayout>

Then i created a view with a listener inside Linear Layout Child that can toggle the expandable layout.

**Comment by yassermohammad** [comment]
Hi, i used this lib, but not work correct.
When i scroll my recyclerview the data is changed and the item that i expand it, collapse automatically!!!

**Comment by Hayko985** [comment]
@cachapa you're closing the issue, but how to fix it? I see you have this problem very long time, so you haven't found a solution yet yes?

**Comment by Hayko985** [comment]
@cachapa but where is the "collapsed_height" attribute, I can't see it.

**Comment by Hayko985** [comment]
@cachapa https://github.com/AAkira/ExpandableLayout . in this library there is a method expandableLayout.setClosePosition(int value); and that is doing what I need, but the library not works good, I had some problems using this library, because I'm using your library. So you can see how he does the collapsed state part, when some part is visible.

**Comment by cachapa** [comment]
@Hayko985 please read my message. The "collapsed_height" attribute doesn't exist yet, all I said is it would be a possible solution for the issue.

If someone wants to implement it, I'll gladly accept it. Otherwise you need to wait until I have the time to do it myself.
